apps/back/src/Controller/ExpenseController.php

```diff
@@ -15,10 +16,22 @@
 #[Route('/expense', name: 'expense')]
 class ExpenseController extends AbstractController
 {
-    #[Route('/list', name: '_list')]
-    public function expensesList(): Response
+    #[Route('/list/{groupName}', name: '_list')]
+    public function expenseList(
+        EntityManagerInterface $em,
+        string $groupName,
+    ): Response
     {
-        return $this->render('expense/list.html.twig', []);
+        $expenseRepository = $em->getRepository(Expense::class);
+        $expenses = $expenseRepository->findBy([], ['date' => 'DESC']);
+
+        $groupRepository = $em->getRepository(Group::class);
+        $group = $groupRepository->findOneBy(['name' => $groupName]);
```

Comment:
`$group` can be null here. You should check whether the value this variable is not null before proceeding to the template rendering.